ckanext/versioning/datapackage.py

```diff
@@ -0,0 +1,29 @@
+"""Frictionless Data datapackage.json related functions
+
+See http://specs.frictionlessdata.io/data-package/ for datapackage specs
+"""
+import frictionless_ckan_mapper.ckan_to_frictionless as ctf
+import frictionless_ckan_mapper.frictionless_to_ckan as ftc
+
+
+def dataset_to_frictionless(package):
+    """Convert a CKAN dataset dict to a Frictionless datapackage
+    """
+    # TODO: Use ckan_mapper
+    #return ctf.dataset(package)
```

Comment:
I assume we are going to update this soon to use new lib - we can always do any "hacking" we need in here. There is a little work we need to do i suspect around setting path @shevron and i worked out the algorithm for that.